observationtree/src/main/scala/explore/observationtree/ObsList.scala

```diff
@@ -129,7 +130,7 @@ object ObsList {
       // Temporary measure until we have id pools.
       val newObs = IO(Random.nextInt()).map(int =>
         ObsSummary(Observation.Id(PosLong.unsafeFrom(int.abs.toLong + 1)),
-                   name.some,
+                   NonEmptyString.from(name).toOption,
```

Comment:
We can move the conversion to `InputModal` by having its `onComplete` callback take a `NonEmptyString` parameter.  (Maybe its `initialValue` can be an `Option[NES]` too).

Reply:
Good idea.